packages/ui/src/SelectField.tsx

```diff
@@ -103,6 +110,27 @@ export const SelectField = <V,>({
     }
   }, [menuPortalTarget])
 
+  // Custom styling
+  //
+  // This is the recommend way of styling react-select:
+  // (https://react-select.com/styles#using-classnames)
+  //
+  // ps. Also it's impossible to style the focused state of
+  // multiValue tags via CSS.
+  const customStyles = {
```

Comment:
@utkukaratas still react-select allows us to use classes. Should we stick to classes and SCSS for consistency?

Reply:
I think @jankoritak did a great job before with Select and demonstrated how it could be easily done with simple class overrides, or component replacements if necessary. WDYT?

Reply:
@aigoncharov css was my first attempt too, but it's impossible to style the focused state of multiValue tags as per our hover-focus designs. i'll show you what i mean with an ss...

Reply:
If we cannot style the existing component for dinner reason, couldn't we replace it with our own one?
https://react-select.com/props#components

Reply:
I'd like to vote for replacement it even if the styling is possible. I find it cleaner.

Reply:
It seems like this one was not addressed yet. Can you look into it, @utkukaratas? Once this is addressed, it seems good for approval.

Reply:
@jankoritak this was an outdated convo. resolving it...

Reply:
Got it. Could you rebase the branch? AFAIC, the PR is good to go.

Reply:
done @jankoritak